pages/apim/3.x/v4/v4-event-native-apim-user-guide.adoc

```diff
@@ -28,28 +28,261 @@ The OpenAPI specification for the management API is available at link:/apim/3.x/
+| No | Yes
+
+| Advanced security (username and password over TLS)
+| No | Yes
+|===
+
+The MQTT connector is useful where, for example, only protocol mediation is needed but guaranteed delivery or security.
+
+The MQTT Advanced connector is useful where messages must be delivered reliably (through quality of service) or securely (using advanced security), such as to customers who would pay for such information.
+
+Using the MQTT connectors provides the following benefits:
+
+* *Reuse existing infrastructure*: for teams moving to MQTT and/or a new MQTT broker, you can now use Gravitee to make it possible for MQTT to communicate with client side applications that talk via REST, Websocket, Webhook, and more.
+* *Make IoT communications more consumer-friendly*: instead of setting up MQTT clients, allow messages coming from MQTT to be consumed by APIs that your consumers are more comfortable with (i.e. Websocket, Webhook, SSE, REST, etc.)
+* *Monetize IoT data streams*: Expose IoT data streams via any consumer-facing API or protocol using the Gravitee Developer Portal and then use Gravitee's monetization capabilities to turn these data streams into revenue streams.
+
+==== Use case
+
+image:{% link /images/apim/3.x/event-native/event-native-api-management-use-case-mqtt.png %}[]
+
+1. An API publisher creates an API with a Websocket entrypoint and a MQTT endpoint.
+2. The API publisher implements the Gravitee Assign metrics policy (EE only) to enable API monetization.
+3. The API publisher publishers the API to their Gravitee Developer Portal. 
+4. Developers discover and subscribe their consumer applications to the API via the Developer Portal.
+5. An event occurs and the message is streamed from the MQTT broker to the Gravitee Gateway. The Gateway then streams them to the proper consumer applications via Websocket connection.
+6. The Assign metrics policy is executed and this allows payment processing systems to invoice API consumers.
+
+=== Kafka
+
+Kafka is a distributed event-streaming platform used for high-performance data pipelines, streaming analytics, and data integration. There are two connectors, Kafka and Kafka Advanced. Kafka Advanced is only available for the Enterprise Edition of the Gravitee Gateway.
+
+The Gravitee Kafka connectors offer advanced protocol mediation capabilities that can be applied for use cases where teams have a Kafka backend and want that broker and backend to communicate with a non-Kafka client. In addition to protocol mediation the advanced connector also supports advanced security feature (user/password using TLS) and quality of service (QoS). 
+
+.Kafka Connectors
+|===
+|Feature | Kafka | Kafka Advanced
+
+| Protocol mediation 
+| Yes | Yes 
+
+| link:#quality_of_service[Quality of Service]
+| No | Yes
+
+| Advanced security (username and password over TLS)
+| No | Yes
+|===
+
+The Kafka connector is useful where, for example, only protocol mediation is needed but guaranteed delivery or security.
```

Comment:
I feel there is a missing word. Maybe it is due to my English level, but when I read that I understand that the regular connector seems to guarantee the delivery/security.

Reply:
Yes, there was a missing word. Thanks for spotting it!